packages/firebase_remote_config/android/build.gradle

```diff
@@ -32,7 +32,6 @@ android {
         disable 'InvalidPackage'
     }
     dependencies {
-        api 'com.google.firebase:firebase-config:[11.4.0,12.0['
-        api 'com.google.firebase:firebase-core:[11.4.0,12.0['
+        api 'com.google.firebase:firebase-config:15+'
```

Comment:
Can you elaborate on why this is necessary, just for my own edification?

Reply:
All libraries from Google Play Services must be of the same version if used in the same app.  We should devise a way to auto update all the plugins so they stay in sync.

Reply:
i mean `15.+`, not `15+`.

Reply:
Good Catch!